manifest.yml

```diff
@@ -0,0 +1,9 @@
+applications:
+- name: ras-backstage
+  disk_quota: 1G
+  health-check-http-endpoint: /info
+  health-check-type: http
+  instances: 1
+  memory: 512M
+  stack: cflinuxfs2
```

Comment:
Stack?

Reply:
All of the configuration was generated from the cloudfoundry command line tool. `cf create-app-manifest`. I can update the manifests to reflect prod if we feel like it's necessary right now but as jenkins doesn't use the manifest I'm not sure it's worth it